src/components/dropdown-card/styles/SDropdownCardBody.tsx

```diff
@@ -1,8 +1,8 @@
-import styled, { css } from 'styled-components';
-import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from '../../../constants/styles/mediaquerys';
+import styled, { css } from "styled-components";
+import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from "../../../constants/styles/mediaquerys";
 
 const Bronze = css`
-padding: 20px;
+  padding: ${(props) => `${props.padding || 20}px`};
```

Comment:
@yousseefben  also here i would not use a padding property - it cleaner if we create a styleOverwrite like in the button and put the padding we need there in web-app

Reply:
okay, i don't need the padding anymore, so i'll rollback